odoo/addons/base/controllers/avatar.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from hashlib import sha1
+import base64
+
+from odoo.http import Controller, request, route, Response, local_redirect
+from odoo.modules.module import get_module_resource
+
+class Avatar(Controller):
+    @route(['/web/avatar/<int:user_id>', '/web/avatar/<int:user_id>/<string:image_field>'], auth='public', cors="*", methods=['GET'])
+    def get_avatar(self, user_id=1, image_field="image_128"):
+        # id 20
+        user = request.env['res.users'].browse(user_id)
```

Comment:
user = request.env['res.users'].browse(user_id).exists()
it will return the record if it exists and a falsy value else.
Otherwise you can browse for a record but it will give you errors.